app/assets/stylesheets/osem-navbar.css.scss

```diff
@@ -0,0 +1,42 @@
+.btn_sign_in {
+    background-color: #24a724;
+    color: white;
+    font-weight: bold;
+    &:hover {
+        color: white;
+        background-color: rgba(41, 147, 11, 1);
+    }
+}
+
+.nav-modify {
+    border: none;
+}
+
+.navbar-collapse.in {
+    overflow: hidden;
+    max-height: none;
+}
+
+.nav-ham-btn .icon-bar {
+    background-color: white !important;
+}
+
+@media (max-width: 768px) {
```

Comment:
Have a look at the [the breakpoints mixin](https://github.com/openSUSE/osem/blob/master/app/assets/stylesheets/breakpoints.css.scss) which makes writing the breakpoint a bit easier.